terraform-module/modules/frontend-spa-s3/main.tf

```diff
@@ -19,7 +19,7 @@ resource "aws_s3_bucket" "origin" {
 
 resource "aws_s3_bucket_acl" "origin" {
   bucket = aws_s3_bucket.origin.id
-  acl    = "private"
+  acl    = null
 }
```

Comment:
Should we just delete this resource all together? Seems a bit hacky to set it to null if we don't want to be setting it at all. What does the plan look like in that case?

Reply:
Makes sense, I'm just doing what a random Github user is recommending! 😄 

If I run the plan locally

When changed to null
```
  # module.app.module.s3.aws_s3_bucket_acl.origin will be updated in-place
  ~ resource "aws_s3_bucket_acl" "origin" {
      - acl    = "private" -> null
        id     = "pleo-product-web-origin-staging,private"
        # (1 unchanged attribute hidden)

        # (1 unchanged block hidden)
    }

```

When I remove the resource
```
# module.app.module.s3.aws_s3_bucket_acl.origin will be destroyed
  # (because aws_s3_bucket_acl.origin is not in configuration)
  - resource "aws_s3_bucket_acl" "origin" {
      - acl    = "private" -> null
      - bucket = "pleo-product-web-origin-staging" -> null
      - id     = "pleo-product-web-origin-staging,private" -> null

      - access_control_policy {
          - grant {
              - permission = "FULL_CONTROL" -> null

              - grantee {
                  - display_name = "admin-aws+product-staging" -> null
                  - id           = "9ad3f0de9704e80636a3d3455c00ba851a7fafb827078b5d2bdeb32d7430e565" -> null
                  - type         = "CanonicalUser" -> null
                }
            }
          - owner {
              - display_name = "admin-aws+product-staging" -> null
              - id           = "9ad3f0de9704e80636a3d3455c00ba851a7fafb827078b5d2bdeb32d7430e565" -> null
            }
        }
    }
```

Does that help?

Reply:
Hmm ok, let's go with `null` for now and figure that other diff out later 😅